### PR TITLE
Ensure dataset/table suggestions work after latest changes

### PR DESCRIPTION
**Pull request by sunker**
**What this PR does / why we need it**:

This PR ensures that autocompletion for tables and datasets still work as expected after [recent changes](https://github.com/grafana/grafana-experimental/pull/43) in the autocompletion engine that added support for prefixing table name with schema. 

bigquery

Please note that since the query editor UI components were recently [moved](https://github.com/grafana/grafana-experimental/commit/bed2f40f506996474660415c779e091dd1d88990) from grafana experimental to grafana ui, I had to upgrade grafana packages to the most recent version. 

**Special notes for your reviewer**:
Since the [source PR](https://github.com/grafana/grafana-experimental/pull/43) is not yet merged to main, there's no new release to point to for the experimental package yet. To test this, one needs to link the experimental package locally.

### CODE REVIEW

**Comment by zoltanbedi** [comment]
Thanks @sunker, I'll try this out asap.

File: src/types.ts [hunk 1]
```diff
@@ -61,6 +60,11 @@ export interface QueryModel extends DataQuery {
   };
 }
 
+export enum EditorMode {
```
Comment (zoltanbedi) [inline src/types.ts:63]: Should we export this from grafana ui?

**Review by zoltanbedi — APPROVED** [review]
Tried this out with the experimental changes and it looked good.

File: src/types.ts (hunk 1 above)
Comment (sunker) [reply src/types.ts:63]: Yeah think it should be. @scottlepp if you're still working on this, can you make sure [EditorMode](https://github.com/grafana/grafana/blob/main/public/app/features/plugins/sql/types.ts#L52) is being exported?

**Comment by sunker** [comment]
Thanks for quick feedback @zoltanbedi! Now this branch is pointing to an actual release of grafana experimental. Everything should be ready for merging and releasing, but I'll leave that for you. Thanks!

**Comment by zoltanbedi** [comment]
Thanks @sunker I'll take care of the rest.

**Comment by zoltanbedi** [comment]
This is good now. I'm going to merge it.

**Pull request merged**